src/cljs/replumb/load.cljs

```diff
@@ -68,32 +111,29 @@
   tries to load the unevaluated ones.
   This function does not check whether parameters are nil, please do it in
   the caller."
-  [verbose? file-names read-file-fn! load-fn-cb cached-file-names]
+  [verbose? file-names read-file-fn! load-fn-cb cached-file-names name]
```

Comment:
@tomasz-biernacki maybe we can pass `opts` here as well if we have it in the caller, so that we avoid rebuilding it below?